packages/components/src/presentation-components/instance-filter-builder/PresentationInstanceFilter.ts

```diff
@@ -499,12 +499,27 @@ function toRelationshipStep(path: RelationshipPath): GenericInstanceFilterRelati
   }));
 }
 
-function toGenericInstanceFilterRuleValue(primitiveValue?: PrimitiveValue): GenericInstanceFilterRuleValue | undefined {
+function toGenericInstanceFilterRuleValue(
+  operator: `${PropertyFilterRuleOperator}`,
+  primitiveValue?: PrimitiveValue,
+): GenericInstanceFilterRuleValue | undefined {
   if (!primitiveValue || primitiveValue.value === undefined || !isGenericPrimitiveValueLike(primitiveValue.value)) {
     return undefined;
   }
 
-  return { displayValue: primitiveValue.displayValue ?? "", rawValue: primitiveValue.value };
+  const displayValue = primitiveValue.displayValue ?? "";
+  let rawValue = primitiveValue.value;
+
+  if (operator === "like" && typeof rawValue === "string") {
+    if (!rawValue.startsWith("%")) {
+      rawValue = `%${rawValue}`;
```

Comment:
How does the round trip works? `PresentationInstanceFilter` -> `GenericInstanceFilter` -> `PresentationInstanceFilter`. From current implementation it looks like if `someValue` was entered into UI then it was converted to `GenericInstanceFilter` and back, UI will be populated with `%someValue%`?

Reply:
You are right. Removed the appended `%` when parsing the filter.

Reply:
I am not sure that this is enough. What about entering `%someValue` into UI and doing same roundtrip?

Reply:
In that case after reopening the filter `someValue` will be displayed, which I think is the expected behavior considering that manually inserting `%` at the beginning or the end has no effect on the filter. Though in that case adding a bunch of them at the start would cause them to be removed one by one instead of removing all of them.

Reply:
I am not sure that this is an expected behavior. I think UI should show same value that was entered without modifying it.